akka-sample-sharding-scala/killrweather-fog/src/main/scala/sample/killrweather/fog/JsonFormats.scala

```diff
@@ -1,11 +0,0 @@
-package sample.killrweather.fog
-
-private[fog] object JsonFormats {
-  // This formatter determines how to convert to and from Data objects:
-  import spray.json.RootJsonFormat
-  // import the default encoders for primitive types (Int, String, Lists etc)
-  import spray.json.DefaultJsonProtocol._
-
-  implicit val weatherStationFormat: RootJsonFormat[WeatherStation.WmoId] = jsonFormat7(WeatherStation.WmoId)
-  implicit val dataFormat: RootJsonFormat[WeatherApi.Data] = jsonFormat3(WeatherApi.Data)
-}
```

Comment:
After removing the CRUD ops for weather stations in the Killrweather cluster, the only thing we need to serialize is the data registration, so made a minimal inline JsObject with that where the request is done instead of having a class in WeatherApi + formats here